infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -315,6 +314,16 @@ public void handleTokenQuery(CorfuPayloadMsg<TokenRequest> msg,
     }
 
     /**
+     * Service an incoming request to mark the server as NOT READY.
+     */
```

Comment:
I think this is something that the sequencer itself should be able to determine. CUrrently, the situations in which a sequencer should mark itself not-ready is when it is started as a replacement sequencer, or after recovery. In the future, we may add more cases.